management/clipper_manager.py

```diff
@@ -67,6 +68,19 @@ class Clipper:
         The SSH port to use. Default is port 22.
     check_for_docker : bool, optional
         If True, checks that Docker is running on the host machine. Default is True.
+    redis_port : int, optional
+        The port to use for connecting to redis. Default is port 6379.
+    redis_ip : string, optional
+        The ip address of the redis instance that Clipper should use.
+        If unspecified, a docker container running redis will be started
+        on `host` at the port specified by `redis_port`.
+    redis_persistence_path : string, optional
+        The directory path to which redis data should be persisted. If the
+        directory does not already exist, it will be created. If unspecified,
+        redis will not persist data to disk. 
```

Comment:
Okay so for some reason providing `/tmp` as redis_persistence_path is breaking docker. As a workaround for now, let's force the user to specify a path that doesn't already exist and verify that in the code.